test/sys/test_resource.rs

```diff
@@ -1,10 +1,5 @@
-#[cfg(not(any(
-    target_os = "redox",
-    target_os = "fuchsia",
-    solarish,
-    target_os = "haiku"
-)))]
```

Comment:
This gate and the below one are removed because we gate them at `test/sys/mod.rs`:

```rs
#[cfg(not(any(
    target_os = "redox",
    target_os = "fuchsia",
    solarish,
    target_os = "haiku"
)))]
mod test_resource;
```